apps/server/lib/google-drive/src/internal/uploads.ts

```
@@ -0,0 +1,175 @@
+export type BlobPart = string | ArrayBuffer | ArrayBufferView | Blob | DataView;
+type FsReadStream = AsyncIterable<Uint8Array> & { path: string | { toString(): string } };
+
+// https://github.com/oven-sh/bun/issues/5980
+interface BunFile extends Blob {
+	readonly name?: string | undefined;
+}
+
+export const checkFileSupport = () => {
+	if (typeof File === "undefined") {
+		const { process } = globalThis as any;
+		const isOldNode = typeof process?.versions?.node === "string" && parseInt(process.versions.node.split(".")) < 20;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)